instructions.texi

```diff
@@ -148,10 +148,19 @@ PC  Byte  Instruction
 Constants Vector: [n]
 @end verbatim
 
-From the above we see that the @code{goto} instruction at program
-counter (PC) 5, has decimal opcode 130. The instruction is three bytes
+From the above we see that the @code{goto} instruction at
+program counter (PC) 5, has decimal opcode 130. The instruction is three bytes
 long: a one-byte opcode followed by a two-byte operand.
 
+The instruction name at PC 0 with opcode 192, @code{constant[0]},
+looks like it is indexing, but it is a just name, where the brackets
+and number are part of the name. We use this kind of name because it is
+suggestive of how it works: it indexes the first element into the
+constants vector and pushes that value onto the evaluation
+stack. @code{constant[1]} with opcode 193 pushes onto the stack. We
```

Comment:
"pushes the second element of the constants vector onto the stack", maybe?

Reply:
Thanks. fixed now.